core/dataset_manager.py

```diff
@@ -54,7 +55,7 @@ def __init__(self, config_manager):
         raw_filepath = config['GENERAL']['dataset_path']
         assert os.path.isdir(raw_filepath), "The dataset filepath provided is not valid."
         self.raw_filepath = raw_filepath
-        self.mappings = None
+        self._mappings = None
```

Comment:
It's good that this is private. Can you make everything that should be private in datasetManager private?

Reply:
also you forgot to change the tests you scrub

Reply:
heckin bamboozled

Reply:
ok now ur actually failing tests

Reply:
no it was more stuff I forgot to update -_-

to quote a good friend of mine:
> I'm too high from all this berkeley smoke